server/index.js

```diff
@@ -11,6 +11,10 @@ app.get('/', (req, res) => {
   res.send('Serving')
 });
 
-app.get('/api/')
+app.get('/api/item', (req, res) => {
+	db.getAllReviews((err, results) => {
+		err ? res.status(404).send(err) : res.send(err, results);
```

Comment:
Your definition of `db.getAllReviews` says that the callback will be given `results` as the first arg, but you seem to be expecting `results as the second arg here. Does this functionality work?

Reply:
it does but there is an error in the console. will fix. thanks for all feedback <3